solidity/contracts/bridge/WalletCoordinator.sol

```diff
@@ -48,6 +48,8 @@ contract WalletCoordinator is OwnableUpgradeable, Reimbursable {
     enum WalletAction {
         /// @dev The wallet does not perform any action.
         Idle,
+        /// @dev The wallet is executing heartbeat.
+        Heartbeat,
```

Comment:
There is some client-side code that uses this enum (e.g. `parseWalletActionType` in `pkg/chain/ethereum/tbtc.go`). Until we reflect that change there, a new enum in the middle will cause a hidden bug as `Heartbeat` will be recognized as `DepositSweep`. Can we add the `Heartbeat` value at the end? This way the client code will just fail on an unrecognized value. This way, we won't forget about updating it on the client-side.

Reply:
I forgot to explain why I decided to keep it first. This was intentional!

The reasoning is: with this order, if we need to upgrade, we will always append after what is already implemented.

Yes, we will have to update the mapping on the Go side but I am planning to integrate `WalletCoordinator`-based signing into the client right after this PR (I'll work on it today). I think the approach where `Heartbeat` and `DepositSweep` are first in the enum makes sense because they will be implemented in the client and any other functionality that we'll be adding in future releases will not require reorganization of the enum or state fields in the upgradeable `WalletCoordinator` contract. Imagine we decide to upgrade the contract and add another `WalletAction` in the future. If `Heartbeat` and `DepositSweep` are at the beginning, we will not have to worry about their values. We will also not have to care about the order of fields in the contract: we will always append after what is already implemented.

Reply:
Ok, makes sense!